icons/unjs.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>UnJS</title><path d="M1.398 0A1.49 1.49 0 0 0 0 1.412V22.59A1.49 1.49 0 0 0 1.398 24H22.6a1.49 1.49 0 0 0 1.4-1.41V1.412A1.49 1.49 0 0 0 22.6 0Zm16.894 15.052c.52-.012 1.004.113 1.399.513q.059.064.115.13c.247.327.353.695.398 1.08.025.21.031.42.032.63.004.884 0 1.768.002 2.652 0 .204-.065.368-.252.47a.445.445 0 0 1-.658-.382c-.004-.213-.002-.428-.002-.64-.001-.78.002-1.557-.004-2.336-.003-.283-.056-.556-.213-.8a.94.94 0 0 0-.713-.433c-.62-.07-1.132.135-1.527.62a1.1 1.1 0 0 0-.239.716q.002 1.419 0 2.837a.46.46 0 0 1-.61.447c-.177-.056-.294-.23-.3-.451v-.258l-.002-4.133c0-.184.056-.342.215-.444a.43.43 0 0 1 .45-.02q.23.122.247.386c.003.035 0 .072 0 .12.03-.024.054-.04.075-.06.388-.352.839-.583 1.363-.63q.113-.012.225-.014m-7.75.15c.235.014.413.225.414.495q.002.825 0 1.648c0 .424-.001.847.006 1.27q.005.318.156.599c.143.259.36.422.646.5.373.102.745.096 1.107-.047.37-.146.578-.435.648-.82.026-.147.032-.3.033-.448q.003-1.35.002-2.703c0-.25.12-.423.338-.48.053-.014.109-.01.164-.014.258.044.409.214.408.47 0 1.024.012 2.047-.017 3.069-.02.697-.334 1.254-.952 1.614a2.2 2.2 0 0 1-.85.27c-.388.044-.777.038-1.154-.071-.598-.174-1.043-.53-1.283-1.122-.126-.311-.16-.638-.162-.968q-.003-1.388 0-2.777c0-.304.205-.502.496-.485"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>UnJS</title><path d="M0 1.32v21.343A1.496 1.496 0 0 0 1.484 24h21.014A1.5 1.5 0 0 0 24 22.501V1.483A1.496 1.496 0 0 0 22.674 0H1.308A1.492 1.492 0 0 0 0 1.321Zm15.719 14.392c0-.092.012-.176.046-.252a.43.43 0 0 1 .166-.19.45.45 0 0 1 .222-.074.45.45 0 0 1 .23.052c.076.04.136.094.178.16.04.062.064.14.07.226v.12l.074-.06c.194-.176.404-.32.63-.428.23-.11.478-.178.731-.2.15-.016.3-.02.444-.012.445.006.869.189 1.178.508l.116.132v.004h.004c.242.324.35.69.394 1.076.026.208.032.42.034.63v2.651c0 .204-.066.368-.252.47a.444.444 0 0 1-.66-.382c-.004-.214 0-.426 0-.64 0-.78 0-1.557-.004-2.335a1.456 1.456 0 0 0-.214-.8.936.936 0 0 0-.712-.432 1.92 1.92 0 0 0-.443 0c-.279.03-.544.136-.766.306a1.886 1.886 0 0 0-.318.312c-.084.104-.144.216-.18.336s-.056.248-.056.38v2.835a.46.46 0 0 1-.612.448c-.088-.028-.162-.086-.214-.164s-.084-.178-.086-.288v-4.39Zm-5.534-.39a.46.46 0 0 1 .356-.12c.236.012.414.224.416.496v1.646c0 .424-.002.847.006 1.271.002.21.055.415.154.598.142.258.36.42.644.498.188.05.374.076.56.068s.368-.042.55-.114c.088-.034.172-.08.248-.136.218-.167.362-.413.4-.686.026-.146.034-.296.034-.446v-2.701c0-.252.12-.424.34-.48.052-.016.108-.012.164-.016h.006c.252.044.402.216.4.472 0 1.022.012 2.046-.016 3.067a1.834 1.834 0 0 1-.952 1.614c-.262.152-.55.234-.85.268-.388.046-.776.04-1.154-.07a2.316 2.316 0 0 1-.42-.164 1.8 1.8 0 0 1-.862-.956 2.568 2.568 0 0 1-.164-.97v-2.775c0-.152.052-.278.14-.364Z"/></svg>
```
@uncenter can you try applying this?

Reply:
![CleanShot 2024-06-16 at 12 15 18@2x](https://github.com/simple-icons/simple-icons/assets/47499684/2ea67d93-ecd7-408a-88e3-3703f2a97ccd)
Still seeing differences in the text, and the top left & bottom left corners are different. Might be a bit better than mine?

Reply:
Yeah, I guess the svgo optimizes it in a way that causes this. I didn't know how to resolve this 😞

Reply:
Hmmm. I think I'll commit yours and since it is the best we can get...

Reply:
```
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>UnJS</title><path d="M22.67 0H1.314A1.487 1.487 0 0 0 0 1.313v21.355C.085 23.418.711 24 1.484 24h21.014A1.5 1.5 0 0 0 24 22.498V1.484C24 .711 23.42.085 22.67 0Zm-4.377 15.05c.52-.011 1.004.113 1.398.514.041.042.077.088.116.131.246.327.353.694.398 1.08.024.209.03.42.031.63.004.883.001 1.768.002 2.652 0 .203-.065.368-.252.468a.445.445 0 0 1-.658-.38c-.004-.214-.002-.428-.002-.641 0-.779.002-1.557-.004-2.336-.002-.282-.056-.556-.213-.799a.937.937 0 0 0-.713-.433c-.62-.071-1.131.133-1.527.617-.17.208-.238.452-.238.718v2.836c0 .33-.297.546-.61.448-.178-.056-.295-.232-.3-.453-.002-.086 0-.172 0-.258l-.002-4.131c0-.184.055-.342.215-.443a.43.43 0 0 1 .449-.022c.155.08.236.213.248.387.002.035 0 .072 0 .12.03-.024.053-.04.074-.06.388-.352.839-.583 1.363-.63.076-.007.15-.012.225-.014zm-7.75.151c.235.014.413.226.414.496v1.647c0 .424-.002.847.006 1.271.004.21.054.412.156.598.142.259.36.423.645.5a1.72 1.72 0 0 0 1.107-.047c.371-.146.58-.434.649-.82.026-.146.032-.299.033-.448.003-.9.001-1.802.002-2.703 0-.25.12-.422.338-.48.052-.014.109-.01.164-.014.258.044.408.214.408.47-.001 1.024.012 2.047-.018 3.07-.02.696-.333 1.254-.95 1.613-.262.152-.55.234-.85.27a2.896 2.896 0 0 1-1.155-.071c-.598-.174-1.044-.53-1.283-1.121-.126-.312-.161-.638-.162-.97-.003-.925-.001-1.85 0-2.776 0-.304.205-.502.496-.485z"/></svg>
```

@uncenter can you try this? i think it's better a bit 😅

Reply:
Looks better on the text! Can you try fixing up the two corners on the left half?

Reply:
```
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>UnJS</title><path d="M22.686 0H1.29A1.477 1.477 0 0 0 0 1.297V22.68C.085 23.43.711 24 1.484 24h21.014A1.5 1.5 0 0 0 24 22.498V1.484C24 .711 23.412.077 22.686 0zm-4.393 15.05c.52-.011 1.004.113 1.398.514.041.042.077.088.116.131.246.327.353.694.398 1.08.024.209.03.42.031.63.004.883.001 1.768.002 2.652 0 .203-.065.368-.252.468a.445.445 0 0 1-.658-.38c-.004-.214-.002-.428-.002-.641 0-.779.002-1.557-.004-2.336-.002-.282-.056-.556-.213-.799a.937.937 0 0 0-.713-.433c-.62-.071-1.131.133-1.527.617-.17.208-.238.452-.238.718v2.836a.46.46 0 0 1-.61.448c-.178-.056-.295-.232-.3-.453-.002-.086 0-.172 0-.258l-.002-4.131c0-.184.055-.342.215-.443a.43.43 0 0 1 .449-.022c.155.08.236.213.248.387.002.035 0 .072 0 .12.03-.024.053-.04.074-.06.388-.352.839-.583 1.363-.63.076-.007.15-.012.225-.014zm-7.75.151c.235.014.413.226.414.496v1.647c0 .424-.002.847.006 1.271.004.21.054.412.156.598.142.259.36.423.645.5.364.107.753.09 1.107-.047.371-.146.58-.434.649-.82.026-.146.032-.299.033-.448.003-.9.001-1.802.002-2.703 0-.25.12-.422.338-.48.052-.014.109-.01.164-.014.258.044.408.214.408.47-.001 1.024.012 2.047-.018 3.07-.02.696-.333 1.254-.95 1.613-.262.152-.55.234-.85.27-.386.054-.779.03-1.155-.071-.598-.174-1.044-.53-1.283-1.121-.126-.312-.161-.638-.162-.97-.003-.925-.001-1.85 0-2.776 0-.304.205-.502.496-.485z"/></svg>
```
Luckily, I still keep the working file. Please try again with this one @uncenter

Reply:
Looks great! Just committed it.